src/reducers/index.js

```diff
@@ -78,16 +91,33 @@ function getInitialState() {
      */
     firstRowIndex: 0,
     firstRowOffset: 0,
+    firstColumnIndex: 0,
+    firstColumnOffset: 0,
     maxScrollX: 0,
     maxScrollY: 0,
     rowOffsets: {},
+    columnOffsets: {},
+    fixedColumnOffsets: {},
+    fixedRightColumnOffsets: {},
     rows: [], // rowsToRender
```

Comment:
This just holds the list of rows that need to be rendered in the table.
I'll rename this to `rowsToRender` on a cleanup review.